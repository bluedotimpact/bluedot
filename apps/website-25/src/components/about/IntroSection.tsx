import { Section } from '@bluedot/ui';

const IntroSection = ({ title }: { title: string }) => {
  return (
    <Section className="intro-section" title={title}>
      <div className="intro-section__container flex lg:flex-row flex-col gap-space-between">
        <div className="intro-section__text-container flex flex-col gap-5">
          <p>Humanity faces unprecedented challenges from transformative AI (TAI) which could profoundly reshape our future. Current AI systems already exhibit concerning behaviours, and as they become more powerful, misalignment between their actions, human intentions and societal values could lead to catastrophic outcomes.</p>
          <p>BlueDot Impact exists to safeguard humanity from these risks by building and accelerating the AI safety field. We've trained over 4,000 professionals and helped hundreds land roles at organisations like Anthropic and the UK's AI Safety Institute. We've built a strong foundation of deep knowledge in how to communicate complex concepts, close relationships across the field, and expertise in talent development. However, urgency and the magnitude of AI risk demands we do much more.</p>
        </div>
        <div className="intro-section__image-container w-full max-w-[570px] aspect-[4/3] relative">
          <img
            className="intro-section__image size-full object-cover rounded-2xl"
            src="/images/team/team_1.jpg"
            alt="BlueDot Impact team"
          />
        </div>
      </div>
    </Section>
  );
};

export default IntroSection;
