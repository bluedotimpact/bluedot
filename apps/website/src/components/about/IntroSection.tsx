import { Section } from '@bluedot/ui';
import { P } from '../Text';

const IntroSection = () => {
  return (
    <Section className="intro-section" title="Our story">
      <div className="intro-section__container flex lg:flex-row flex-col gap-space-between">
        <div className="intro-section__text-container flex flex-col gap-5">
          <P>In 2021, we and others around us wanted to learn about emerging technologies and how they might reshape the world â€“ but nobody knew where to start. It was hard to find what to read, people to discuss it with, and ways to direct our careers to making a difference. The field of AI was moving so fast, and most of the key literature was in blog posts and arXiv pre-prints, rather than textbooks and peer-reviewed journals. Traditional education institutions were struggling to keep up.</P>
          <P>What started as small study groups at the University of Cambridge has grown into something much bigger: courses designed with some of the world's leading experts that have helped thousands of people understand and work on these challenges.</P>
          <P>As technology continues to advance rapidly, we're constantly adapting what we do. We work closely with leading organisations to understand what skills they need, quickly build new courses, and help talented people find roles where they can make a real difference.</P>
          <span className="intro-section__signature">
            <P>Dewi and Will,</P>
            <P>BlueDot Impact Co-Founders</P>
          </span>
        </div>
        <img className="intro-section__image max-w-[570px] w-full h-auto max-h-[400px] object-cover rounded-md mt-6 sm:mt-0 shrink-0" src="/images/culture/About-Us_v1.jpg" alt="BlueDot Impact team" />
      </div>
    </Section>
  );
};

export default IntroSection;
