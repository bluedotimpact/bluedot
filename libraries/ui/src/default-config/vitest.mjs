/* eslint-disable import/no-extraneous-dependencies */
import { defineConfig, defaultExclude } from 'vitest/config';
import react from '@vitejs/plugin-react';
import * as dotenv from 'dotenv';

/**
 * @param {import('vitest/config').UserConfig} [config] 
 * @returns {import('vitest/config').UserConfig}
 */
export const withDefaultBlueDotVitestConfig = async (config) => defineConfig({
  ...config,
  plugins: [react(), ...(config?.plugins ?? [])],
  test: {
    environment: 'happy-dom',
    env: dotenv.config({ path: '.env.test' }).parsed,

    ...config?.test,

    exclude: [...defaultExclude, '**/.{turbo,next}/**', ...(config?.test?.exclude ?? [])],
  },
});
