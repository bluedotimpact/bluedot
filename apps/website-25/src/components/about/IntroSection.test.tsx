import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import IntroSection from './IntroSection';

describe('IntroSection', () => {
  test('renders default as expected', () => {
    const { container } = render(<IntroSection />);
    expect(container).toMatchSnapshot();
  });
});
