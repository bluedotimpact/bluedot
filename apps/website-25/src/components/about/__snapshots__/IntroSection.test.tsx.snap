// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IntroSection > renders default as expected 1`] = `
<div>
  <div
    class="section w-full max-w-max-width min-w-min-width mx-auto overflow-y-scroll md:px-gutter px-gutter-sm my-8 intro-section"
  >
    <div
      class="section__title-container flex justify-between items-center gap-4"
    >
      <div
        class="section__content flex-1"
      >
        <h1
          class="section__title mb-4 relative after:w-full"
        >
          Hello, World
        </h1>
      </div>
    </div>
    <div
      class="section__body"
    >
      <div
        class="intro-section__container flex lg:flex-row flex-col gap-4"
      >
        <div
          class="intro-section__text-container flex flex-col gap-5"
        >
          <p>
            Humanity faces unprecedented challenges from transformative AI (TAI), which could profoundly reshape our future. Current AI systems already exhibit concerning behaviours, and as they become more powerful, misalignment between their actions, human intentions and societal values could lead to catastrophic outcomes.
          </p>
          <p>
            BlueDot Impact exists to safeguard humanity from these risks by building and accelerating the AI safety field. Thus far, we’ve trained over 3,000 professionals and helped hundreds land roles at organisations like Anthropic and the UK’s AI Safety Institute. We’ve built a strong foundation, including deep knowledge of how to communicate complex concepts, close relationships across the field, and expertise in talent development. However, the urgency and magnitude of AI risks demand we do much more.
          </p>
        </div>
        <img
          alt="BlueDot Impact team"
          class="intro-section__image max-w-[570px] w-full h-auto rounded-2xl"
          src="/images/team/team_1.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
